CIP-tx-serialization-deprecation-cycles/README.md

```diff
@@ -0,0 +1,116 @@
+---
+CIP: 80
+Title: Transaction Serialization Deprecation Cycle
+Authors: Jared Corduan <jared.corduan@iohk.io>
+Status: Proposed
+Category: Ledger
+Created: 2022-11-09
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This CIP specifies a policy for the backwards compatibility of the serialization scheme of
+Cardano transactions.
+
+## Motivation
+
+Transactions on Cardano are sent on the wire using CBOR and are specified with CDDL.
+The first scheme was introduced with the Byron phase.
+This scheme was changed dramatically with the introduction of the Shelley phase.
+As of the time of the writing of this CIP, however, every new scheme has been backwards
+compatible with the original scheme from the Shelley phase.
+The intention is still to maintain backwards compatibility to the extent reasonable,
+and to make explicit our policy for breaking backwards compatibility when deemed necessary.
+
+## Specification
+
+Problems with serialization fall into two categories:
+* flaws in the implementation
+* flaws is the CDDL specification
+
+Note that at the time of the writing of this CIP, there is only one implementation of the Cardano
+node, and we do not yet need to consider inconsistencies between different implementations.
+
+The policy for maintaining backwards compatibility with the transaction serialization will be
+as follows.
+
+### Serious Flaws
+
+A **serious flaw** in the serialization is an issue which could have a large and negative impact
+on the network, and which requires a hard fork to fix.
+These will almost always be problems with the serialization and not the specification.
+It is up to human discretion to determine what constitutes a serious flaw,
+mostly likely by the core developers.
+
+Backwards compatibility can be abandoned in the case of a serious flaw,
+and **the fix will occur at the next available hard fork**.
+
+### Non-Serious Flaws
+
+A **non-serious flaw** in the serialization is an issue which is not safety critical,
+but is problematic enough to merit breaking backwards compatibility and requires a
+hard fork to fix.
```

Comment:
Here is an interesting question: what about "Non-Serious Flaws" that can be fixed by a soft fork, such as https://github.com/input-output-hk/cardano-ledger/issues/3240.

Block producers must upgrade for both hard and soft forks, but in the case of a soft fork, nodes that are only validating do not have to upgrade. So I think this is correct as written, and we should be explicit about being able to upgrade more rapidly in the case of soft forks.